plugins/woocommerce/src/Blocks/Shipping/ShippingController.php

```
@@ -558,7 +571,8 @@ function ( $rate ) {
 	 * @return bool
 	 */
 	public function track_local_pickup( $served, $result, $request ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$result'. (undefined)

(UnusedFormalParameter)